src/Formatter/LogzIoFormatter.php

```diff
@@ -30,7 +30,7 @@ public function __construct(int $batchMode = self::BATCH_MODE_NEWLINES, bool $ap
      *
      * @param array $record
      *
-     * @see https://support.logz.io/hc/en-us/articles/210206885-How-can-I-get-Logz-io-to-read-the-timestamp-within-a-JSON-log-
+     * @link https://support.logz.io/hc/en-us/articles/210206885-How-can-I-get-Logz-io-to-read-the-timestamp-within-a-JSON-log-
```

Comment:
Why? Please can you explain why is that better.

Reply:
"better" is the wrong word here, its mostly used for internal references between methods and *can* be used for links:
```php
  * @see MyClass::$items           For the property whose items are counted.
  * @see MyClass::setItems()       To set the items for this collection.
```

but `@link` is explicitely only for external [references](http://docs.phpdoc.org/references/phpdoc/tags/link.html)

Reply:
Thanks for clarify!

Reply:
Welcome 😊👍🏻